karapace/kafka_rest_apis/__init__.py

```diff
@@ -35,13 +35,13 @@ class FormatError(Exception):
 
 class KafkaRest(KarapaceBase):
     # pylint: disable=attribute-defined-outside-init
-    def __init__(self, config_path: str) -> None:
-        super().__init__(config_path)
+    def __init__(self, config_file_path: str, config: dict) -> None:
```

Comment:
I wanted to remove `config_file_path` and have only `config`. But the `KarapaceBase` needs it because a few tests rely on the availability of the config path.